client/src/components/login-signup/Login.js

```diff
@@ -1,22 +1,45 @@
-import React from "react";
+import React, { useState, useContext } from "react";
+import AuthService from '../../Services/AuthService';
+import Message from '../Message/Message';
+import {AuthContext} from '../../Context/AuthContext';
 import { useForm } from "react-hook-form";
 import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 
-export default function Login() {
+export default function Login(props) {
   const { register, handleSubmit } = useForm();
```

Comment:
What is does this useForm hook do? Would be useful to have comments explaining what it does

Reply:
there is lots of inbuilt methods we can use like register, handleSubmit, reset etc..